src/js/utils/dates.js

```diff
@@ -0,0 +1,11 @@
+export const setHoursWithOffset = (date) => {
```

Comment:
It looks like lines 89-95 of Calendar.js could be removed and we could just call this function instead

Reply:
Good call! 
Done